src/glibc/sysdeps/x86/bits/setjmp.h

```diff
@@ -27,13 +27,13 @@
 
 #ifndef _ASM
 
-# if __WORDSIZE == 64
+// # if __WORDSIZE == 64
```

Comment:
lets not commit commented code, add a comment here that this was removed and why